packages/create-gator-app/templates/nextjs/starter/src/components/RedeemDelegationButton.tsx

```diff
@@ -36,7 +36,7 @@ export default function RedeemDelegationButton() {
       account: smartAccount,
       calls: [
         {
-          to: getDeleGatorEnvironment(chainId).DelegationManager,
+          to: getDeleGatorEnvironment(chain.id).DelegationManager,
```

Comment:
You can do this, to avoid having to resolve the environment yourself.

```suggestion
          to: smartAccount.environment.DelegationManager,
```